app/components/autocomplete_select_form_component.rb

```diff
@@ -0,0 +1,60 @@
+class AutocompleteSelectFormComponent < ApplicationComponent
+  attr_reader :model, :scope, :url, :data, :input
+
+  # data: { :turbo, :controller, :autocomplete_path_value,
```

Comment:
Are these meant to be here?

Reply:
I figured they would be good to keep in, as useful information for developers using the component.